File: app/src/main/java/com/devmaster/conversordemoeda/MainActivity.java
package com.devmaster.conversordemoeda;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity implements View.OnClickListener {

    private ViewHolder mViewHolder = new ViewHolder();

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        this.mViewHolder.editValue = findViewById(R.id.edit_value);
        this.mViewHolder.editDolar = findViewById(R.id.text_dolar);
        this.mViewHolder.editEuro = findViewById(R.id.text_euro);
        this.mViewHolder.buttonCalculate = findViewById(R.id.button_calculate);

        this.mViewHolder.buttonCalculate.setOnClickListener(this);

    }

    @Override
    public void onClick(View v) {

    }

    private static class ViewHolder {
        EditText editValue;
        TextView editDolar;
        TextView editEuro;
        Button buttonCalculate;
    }

}

File: app/src/main/java/com/devmaster/conversormoedas/MainActivity.kt
package com.devmaster.conversormoedas

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.EditText

public class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main);

        EditText valor = findViewById(R.id.editValor);
        valor.getText();
    }

}
